fedimint-server/src/net/peers.rs

```diff
@@ -466,7 +466,9 @@ where
                 match message_res {
                     Ok(peer_message) => {
                         if let PeerMessage::Message(msg) = peer_message {
-                            self.incoming.try_send(msg).ok();
+                            if self.incoming.try_send(msg).is_err(){
+                                debug!(target: LOG_NET_PEER, "Could not relay incoming message: receiver dropped");
+                            }
```

Comment:
This also happens when the channel for incoming and outgoing messages is full not just when the receiver is dropped - I have updated the logs to reflect this.

Reply:
It might become annoying if we want to debug a federation with one offline peer, but I don't have a good solution. Ideally we could rate limit logs of the same type (this one will supposedly be very spammy).